app/Controllers/feedController.php

```diff
@@ -514,13 +516,11 @@ public static function actualizeFeeds(?int $feed_id = null, ?string $feed_url =
 					// Feed is cached and unchanged
 					$newGuids = [];
 					$entries = [];
-					$feedIsEmpty = false;	// We do not know
```

Comment:
Please avoid mixing unrelated things. This is not related to PHPDoc,
So not for that PR, but even though it was already declared before, it was more explicit this way (or at least a comment should be kept).